test/integration/job/job_test.go

```diff
@@ -2985,3 +3004,32 @@ func updateJob(ctx context.Context, jobClient typedv1.JobInterface, jobName stri
 	})
 	return job, err
 }
+
+func addFinalizerAndDeletePods(ctx context.Context, t *testing.T, clientSet clientset.Interface, namespace string) {
+	t.Helper()
+	pods, errList := clientSet.CoreV1().Pods(namespace).List(ctx, metav1.ListOptions{})
```

Comment:
To keep the variable names consistent:
```suggestion
	pods, err := clientSet.CoreV1().Pods(namespace).List(ctx, metav1.ListOptions{})
```